Docs terminology cleaning (#78)
onflow/flow: docs/content/token/staking/node-operators.mdx

you can [enable delegation for your node](custody-staking-and-delivery#delegation), which allows any user to commit stake for your node 
and receive a proportional percentage of your node's rewards, as determined by you.

## Unstaking / Withdrawing Stake and Rewards

Stake can be [unstaked / withdrawn](custody-staking-and-delivery#unstaking-withdrawing-stake)
after a week-long lock-up.

> Rewards are deposited into the same pool of unstaked tokens where unstaked tokens are stored.

Reviewer: ```suggestion
Rewards are deposited into their own pool of reward tokens.
```

Reviewer: This whole page needs to be edited, but I saw this so I figured I'd change it now.
